completions/bash/cou_bash_completion

```diff
@@ -0,0 +1,61 @@
+_cou_bash_completion() {
+    COMPREPLY=()
+    cur="${COMP_WORDS[COMP_CWORD]}"
+
+    # Define top level options
+    top_opts="--help --version"
+
+    # Define the available subcommands
+    subcommands="plan run help"
+
+    # Define completion for top-level options and subcommands
+    if [ $COMP_CWORD -eq 1 ]; then
+        COMPREPLY=( $(compgen -W "${subcommands} ${top_opts}" -- "${cur}") )
+        return 0
+    fi
+
+    # Define subcommand-specific options and child commands
+    if [ $COMP_CWORD -eq 2 ]; then
+        cmd="${COMP_WORDS[1]}"
+        case "${cmd}" in
+            plan)
+                opts="--help --quite --verbose --parallel --model"
+                child_commands="control-plane data-plane"
+                ;;
+            run)
+                opts="--help --quite --verbose --parallel --model --interactive --no-interactive --i-really-mean-it"
```

Comment:
```suggestion
                opts="--help --quiet --verbose --parallel --model"
                child_commands="control-plane data-plane"
                ;;
            run)
                opts="--help --quiet --verbose --parallel --model --interactive --no-interactive --i-really-mean-it"
```

Reply:
I'd also suggest dropping `--i-really-mean-it` (regardless of whether you agree on dropping the actual option): it'd be very unfortunate if someone caused downtime by mistake because they tabbed too hard ;)

Reply:
Done